platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -22,12 +22,11 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark Responding to Map Position Changes
 
 /**
- Tells the delegate that the viewpoint depicted by the map view is about to
- change.
-
- This method is called whenever the currently displayed map camera will start
- changing for any reason.
-
+ Tells the delegate that the viewpoint depicted by the map view is changing.
+ 
+ This method is called whenever the currently displayed map camera has finished
+ changing, after any calls to `-mapViewRegionIsChanging:` due to animation.
```

Comment:
This is incorrect: this method is called _before_ any calls to `-mapViewRegionIsChanging:`.